includes/wpml/class-sensei-wpml.php

```diff
@@ -34,4 +57,73 @@ public function sensei_after_sending_email() {
 		*/
 		do_action( 'wpml_restore_language_from_email' );
 	}
+
+	/**
+	 * Set language details for the lesson when it is created.
+	 *
+	 * @since $$next-version$$
+	 * @param int $lesson_id Lesson ID.
+	 * @param int $course_id Course ID.
+	 */
+	public function set_language_details_when_lesson_created( $lesson_id, $course_id ) {
+
+		// Get course language_code.
+		$language_code = apply_filters(
+			'wpml_element_language_code',
+			null,
+			array(
+				'element_id'   => $course_id,
+				'element_type' => 'post_course',
```

Comment:
I take it WPML prefixes entries with `post_`?

Reply:
Good catch, exactly for this hook it is a CPT slug without prefixes: https://wpml.org/wpml-hook/wpml_element_language_code/

For others it contains that prefix: https://wpml.org/wpml-hook/wpml_set_element_language_details/

But I'm checking every hook.

Thank you!